docs/frequently-asked-questions.rst

```diff
@@ -542,12 +542,25 @@ contract level) with ``arrayname.length = <some new length>;``. If you get the
 
 ::
 
-    int8[] memory memArr;        // Case 1
-    memArr.length++;             // illegal
-    int8[5] storageArr;          // Case 2
-    somearray.length++;          // legal
-    int8[5] storage storageArr2; // Explicit case 2
-    somearray2.length++;         // legal
+    pragma solidity ^0.4.18;
+
+    contract C {
```

Comment:
You have to add a special comment here that this snippet does not compile. As part of our tests, we automatically try to compile all source snippets in the documentation.